SetupDataPkg/Tools/KnobService.py

```diff
@@ -972,9 +972,7 @@ def generate_getter_implementation(schema, header_path, efi_type):
         out.write("#pragma pack ()" + get_line_ending(efi_type))
         out.write(get_line_ending(efi_type))
 
-        out.write("STATIC CHAR8 CachedPolicy[CACHED_POLICY_SIZE + CACHED_POLICY_HEADER_SIZE];")
-        out.write(get_line_ending(efi_type))
-        out.write("STATIC BOOLEAN CachedPolicyInitialized = FALSE;")

```

Comment:
I didn't see any potential usage of `CachedPolicyInitialized`. Since this is generated code and names might be mangled together into that name through conditional logic in some cases, it would be nice for confirmation its not expected to be used.

Reply:
This boolean should be gone. Thanks for removing that.